ex4/server/clients/pokemon_client.js

```diff
@@ -0,0 +1,19 @@
+const axios = require('axios');
+
+class PokemonClient {
+	constructor() {
+		this.API_BASE = 'https://pokeapi.co/api/v2/pokemon';
+	}
+
+	async getPokemon(pokemonId) {
+		try {
+			const response = await axios.get(`${this.API_BASE}/${pokemonId}`);
+
+			return response.data;
+		} catch (error) {
+			throw error;
+		}
```

Comment:
why catching if you just throwing? you can log the error and then throw it again